vignettes/MDA.Rmd

```diff
@@ -15,22 +15,39 @@ knitr::opts_chunk$set(
+
 ```
 
-# Parameterisation
+## Introduction
+
+Malaria chemoprevention encompasses a suite of strategies for reducing malaria morbidity, mortality, and transmission through the treatment of at-risk populations with antimalarial drugs. Three such strategies are mass drug administration (MDA), seasonal malaria chemoprevention (SMC), and perennial malaria chemoprevention (PMC). MDA attempts to simultaneously treat an entire population, irrespective of their infection risk or status. SMC targets children aged 3-59 months with a course of an antimalarials through the peak malaria transmission season. PMC is designed for settings with perennially high transmission and seeks to administer a full antimalarial course at defined intervals irrespective of age or disease status.
+
+`malariasimulation` enables users to simulate the effects of chemoprevention strategies on malaria transmission, and provides the flexibility to design and simulate a range of MDA/SMC/PMC campaigns for a suite of antimalarial drugs. Specifically, users are able to load or specify parameters describing both the characteristics of the antimalarial(s) (e.g. drug efficacy, prophylactic profile) and their distribution (e.g. coverage, distribution timing, and the section of the population to treat). This vignette provides simple, illustrative demonstrations of how MDA and SMC campaigns can be set-up and simulated using `malariasimulation`. The functionality to simulate PMC campaigns is also built into `malariasimulation`, but we do not demonstrate that here.
+
+## Mass Drug Administration
+
+To illustrate how MDA can be simulated using `malariasimulation`, we will implement a campaign that distributes a single dose of sulphadoxine-pyrimethamine amodiaquine (SP-AQ) to 80% of the entire population once a year over a two year period, initiated at the end of the first year.
 
-We are going to set the default parameters to run the simulation from an equilibrium.
+### Parameterisation
+
+The first step is to generate the list of `malariasimulation` parameters using the `get_parameters()` function. Here, we'll use the default `malariasimulation` parameter set (run `?get_parameters()` to view), but specify the seasonality parameters to simulate a bimodal rainfall pattern (illustrated in the plot of adult female mosquito population dynamics below). To do this, we set `model_seasonality` to TRUE within `get_parameters()`, and then tune the rainfall pattern using the `g0`, `g`, and `h` parameters (which represent fourier coefficients) to generate the seasonal rainfall profile. In the `malariasimulation` model, rainfall governs the time-varying mosquito population carrying capacity through density-dependent regulation and, therefore, annual patterns of malaria transmission. Next, we use the `set_equilibrium()` function to tune the initial human and mosquito populations to those required to observe the specified initial EIR.
```

Comment:
I feel like using the bimodal seasonality profile complicates things here, especially wrt SMC. Can we change to a more simple  single peak seasonal profile?

Reply:
Here are the params I use for a highly seasonal setting: 
```
seas_name <- "highly seasonal"
seasonality <- list(c(0.284596,-0.317878,-0.0017527,0.116455,-0.331361,0.293128,-0.0617547))
```

Reply:
The examples have been re-run with the highly seasonal unimodal rainfall setting and, where necessary, the textual explanations updated to the reflect this change.